### ISSUE TITLE
BGP Community Juniper/Junos

### ISSUE THREAD

**Issue by brevius** [checktheroads/hyperglass#37]
Hi,
I noticed thet query for community like ```65535:666``` gives following output
```
^
invalid value '666' in ip address: '666' at '666'

                                                                   ^
invalid value '666' in ip address: '666' at '666'
```
while querying for 65000:1 results in following commands on the router:
```
show route protocol bgp table inet.0 community 65000 1 
show route protocol bgp table inet6.0 community 65000 1 
```
Next - the query for large community like ```200000:101:0``` gives following output:
```
^
syntax error.

                                                                      ^
syntax error.
```
However on Junos I think we should query for smething like ```large:200000:101:0```

**Comment by checktheroads** [comment]
Thank you for opening this - I had noticed it myself on my org's instance. It's due to some overzealous parsing of an AS_PATH query for JunOS - this parsing needs to be limited to `bgp_aspath` query and not applied to everything. I'll get this addressed asap.

**Comment by checktheroads** [comment]
This is fixed in 1.0.0-beta30.
